README.md

```diff
@@ -156,6 +160,54 @@ There is a know issue with session restoration on re-open though. [See GitHub is
 
 ![Firefox settings](./screenshots/firefox_config.png)
 
+### FireFox Features
+
+#### Theme Settings
+
+The Doki Theme for FireFox can synchronize with your device theme settings.
+
+Never want to see light mode ever again? No problem! The *Options page* now allows you to choose
+between `light`, `dark`, `device`, `druthers`, or `all` *(all themes)*. So take your time and
+browse.
+
+**Match Device**
+
+`device` will load only themes based on your current device theme settings *(either light or dark)*.
+In other words, your device settings determines what characters are present in the popup menu.
+
+#### Random Select *(Dark System Setting)*
+https://user-images.githubusercontent.com/23222943/146148772-c7e236ed-6be6-46b6-ac2d-aa3f1b1326ca.mp4
+
+#### Mixed Mode *(Light System Setting)*
+https://user-images.githubusercontent.com/23222943/146149663-22ad428e-0cc7-45f8-9700-1fc4c6f616d2.mp4
+
+**Druthers**
+
+`druthers` is similar to `device` except the character chosen for the light theme and the
+character chosen for the dark theme will alternate depending on your current device settings.
+
+For clarity, if I choose *Asuna* for the **light theme** & *Hatsune Miku* for the **dark theme**.
+If the device setting is **currently dark**, then the **dark** *Hatsune Miku theme* will show. If the device
+setting is *light*, then the **light** *Asuna* theme will show.
+
+#### Example (Maple *(light)* / Jahy *(dark)*)
+https://user-images.githubusercontent.com/23222943/146766085-ecb1b416-a678-4cbc-a75c-44c7fb5b32de.mp4
```

Comment:
Another visual example conveying the `druthers` feature.

Reply:
Yup, that's legit. I did change the documentation to reflect the content's of the video. Felt a bit weird to give a written example & a visual example that where different.